packages/aws-cdk/test/integ/cli/bootstrapping.integtest.ts

```diff
@@ -1,52 +1,175 @@
-import { cloudFormation, deleteStacks } from './aws-helpers';
-import { cdk, cleanupOldStacks, fullStackName, prepareAppFixture } from './cdk-helpers';
+import { cloudFormation, deleteStacks, s3 } from './aws-helpers';
+import { cdk, cdkDeploy, cleanupOldStacks, deleteableStacks, fullStackName, prepareAppFixture } from './cdk-helpers';
 
-jest.setTimeout(600 * 1000);
+jest.setTimeout(600_000);
+
+// Needs to not start with the ACTUAL fullstack name otherwise cleanupOldStacks will pick it up.
+const BOOTSTRAP_STACK_PREFIX = `toolkit-${fullStackName('stack')}`;
+const BUCKETS_TO_DELETE = new Array<string>();
+const QUALIFIER = randomString();
 
 beforeAll(async () => {
   await prepareAppFixture();
 });
 
 beforeEach(async () => {
   await cleanupOldStacks();
+  BUCKETS_TO_DELETE.splice(0, BUCKETS_TO_DELETE.length);
```

Comment:
Can we just turn `BUCKETS_TO_DELETE` into `let bucketsToDelete: string[]`, and here do `bucketsToDelete = []`?